movieNormal.sql

```diff
@@ -0,0 +1,90 @@
+--Select all columns and rows from the movies table
+SELECT *
+FROM movies;
+
+--Select only the title and id of the first 10 rows
+SELECT title, movieid
+FROM movies LIMIT 10;
+
+--Find the movie with the id of 485
+SELECT *
+FROM movies
+WHERE movieid = 485;
+
+--Find the id (only that column) of the movie Made in America (1993)
+SELECT movieid
+FROM movies
+WHERE title LIKE 'Made in A%';
```

Comment:
This will match too many titles.  If there is a movie called Made in Amsterdam it will find that as well.